packages/maskbook/src/components/InjectedComponents/AutoPasteFailedDialog.tsx

```diff
@@ -116,12 +116,23 @@ export function AutoPasteFailedDialog(props: AutoPasteFailedDialogProps) {
                             </Button>
                         ) : null}
                         {url ? (
-                            <Button
-                                variant="text"
-                                onClick={() => saveAsFileFromUrl(url, fileName)}
-                                startIcon={<Download />}>
-                                {t('download')}
-                            </Button>
+                            process.env.architecture === 'app' && process.env.target === 'firefox' ? (
```

Comment:
extract to a common component, it is reusable.

e.q: `packages/maskbook/src/utils/components/DownloadButton.tsx`

CC @Jack-Works

Reply:
Hi, this is temporary workaround, this part will be simplified later.

Reply:
add an `// HACK` comments, thanks.

Reply:
ok

Reply:
ACK